src/main/java/com/amazon/opendistroforelasticsearch/security/support/wildcard/Wildcard.java

```diff
@@ -0,0 +1,70 @@
+package com.amazon.opendistroforelasticsearch.security.support.wildcard;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+// Case-sensitivity and other options are defined by concrete Wildcard implementation
+public interface Wildcard {
+    boolean matches(String candidate);
+
+    default boolean matchesAny(Collection<String> candidates) {
```

Comment:
`matchesAny` -> `matchAny` to be consistent with `WildcardMatcher`.

Reply:
Ok